Resources/Prototypes/_NF/Catalog/Fills/Items/weapon_cases_expedition.yml

```diff
@@ -233,6 +375,33 @@
     - id: CartridgeRocket
       amount: 2
 
+- type: entity
+  parent: WeaponCaseLong
+  id: WeaponCaseLongLauncherChinaLake
+  suffix: Dungeon, China Lake
+  noSpawn: true
+  components:
+  - type: StorageFill
+    contents:
+    - id: WeaponLauncherChinaLake
+      amount: 1
+    - id: GrenadeFrag
+      amount: 2
+
+# LMG
+- type: entity
+  parent: WeaponCaseLong
+  id: WeaponCaseLongLauncherL6
```

Comment:
`LightMachineGunL6` instead of `LauncherL6`?  Remember to change other instances if you do change this.

Reply:
Done